### ISSUE TITLE
Adding a local custom filters' list

### ISSUE THREAD

**Issue by mazesy** [gorhill/uBlock#3669]
Hi,

Is there any way to add a local custom filters' list?
I tried `file:///D:/1/2/AntiPubWeb.txt` but uBO reports the link as broken
I tried `url(file:///D:/1/2/AntiPubWeb.txt)` but uBO doesn't even add it.

I have a 300KB filters' list I have archived after the issuer disappeared (broken url since months ago).
At this time I've added all the filters to _My filters_, but it's bothering to have those 300KB among those filters.

Thanks.

**Comment by WowDude** [comment]
Hi, you didn't even mention what browser you use, but probably Firefox, and if so, then it's a duplicate of https://github.com/gorhill/uBlock/issues/2922 and https://github.com/gorhill/uBlock/issues/3132

**Comment by mazesy** [comment]
Thanks @WowDude 
I didn't mention the browser, the context because I believed it asn't pertinent to my question.

**Comment by uBlock-user** [comment]
Make sure `Allow access to file URLs` is ticked for uBlock Origin.

**Comment by mazesy** [comment]
> Make sure `Allow access to file URLs `is ticked for uBlock Origin.

@uBlock-user I've been a uBO user practically ever since it was launched yet I don't see what you are referring to.

**Comment by uBlock-user** [comment]
@marey I've been a uBO user practically ever since it was launched yet I don't see what you are referring to.

![](https://i.gyazo.com/02aa1907e2d5e17a2324162225a5d87c.png)

By default it's NOT ticked in Chromium, the setting may be different in FF.

**Comment by WowDude** [comment]
@uBlock-user probably you have missed that @mazesy edited [his comment](https://github.com/gorhill/uBlock/issues/3669#issuecomment-378284909) and mentioned already he uses Firefox.

> @mazesy :
> `http://localhost/yourlist.txt` => `http://localhost/D://1/2/AntiPubWeb.txt` doesn't make it.
> 
> So complicated to get one local file included in my filters' list. I do know now that it's specific to Firefox (55+ was it?) that I think I'll just stick on to the list included in My filters. I've spent too much time trying to get the simplest among the simplest to work.

Some extra effort, yes, but not so complicated, I never had any problems with loading localhost filter lists into uBO via local server, you're just doing it wrong, try: `http://localhost/AntiPubWeb.txt` or `http://localhost/1/2/AntiPubWeb.txt`, if still none works, that means you are not familiar with local http server at all and should go read [the server manual](http://webdesy.com/how-to-setup-a-local-server-using-wamp/) (WAMP in this case), especially the "What’s next?" part.

**Comment by mazesy** [comment]
@uBlock-user , first: thanks for detailing. The idea is that a local server _has_ to be set up, which explains it all. OK. Now I have the elements. I am indeed not at all familiar with local http server. It appears an extra step is necessary, the process of including a local filters' list is not _natural_ so-to-say. Thanks as well for the links. Now we're advancing.

**Comment by uBlock-user** [comment]
@WowDude I didn't miss it, I mentioned it may be different for Firefox, I'm not a FF user FYI. Chromium/Chrome require permission to allow access.

**Comment by mazesy** [comment]
@uBlock-user maybe did @WowDude consider you hadn't taken into consideration the fact Firefox was the browser when you wrote 

> Make sure Allow access to file URLs is ticked for uBlock Origin.

Anyway thanks to both of you.

**Comment by WowDude** [comment]
@mazesy 

Exactly, I was refering to uBlock-user 's first comment, perhaps I should mention that, anyway I forgot/missed (despite the fact that I read it) that he corrected himself in his next comment , so I shouldn't refer anymore to his 1st obsolete comment, so it seems that I missed something too.

> @uBlock-user : Chromium/Chrome require permission to allow access.

You are right, I just reinstalled uBO on Chrome/Opera and indeed the option is not ticked by default on Chrome, but in ChrOpera it is. As I installed Chrome just several days ago I was sure that I didn't touch the option, I don't remember touching it, but memory sometimes fails, as in this case.
